### DISCUSSION TITLE
Having a problem with steam start_protected_game.exe

### DISCUSSION THREAD

**Discussion by jheavejimenez** [The-Wineskin-Project/WineskinServer#416] [Q&A]
<img width="1098" alt="image" src="https://github.com/Gcenx/WineskinServer/assets/40889884/ceb3353f-273d-41f2-9bd9-a24693dfc0ec">

Hi, i'm trying to run farlight 84 in steam and i got this problem. does anyone have encountered this problem? I assume that the games running the start_protected_game.exe first will encounter this problem. etc elden ring

by the way i'm running the wineskin on macbook pro m1 chiptset

**Comment by Gcenx** [comment]
Usually the executable with that game have some form of DRM that doesn’t play nicely with wine, you’d need to launch the DRM executable if they provide one.

**Reply by jheavejimenez** [reply]
What is DRM, and how do i execute that?
